### PR TITLE
[Feat.] Send Video Audio to users.

### PR DESCRIPTION
**Pull request by vineetvk01**
Feature :tada: 
1. User will be able to send ( <input "accepts" >) : 
     a. Video: _video/mp4_
     b. Audio: _audio/x-m4a_ (used for recordings)

Preview:
multi-media

Closes:#1094

### CODE REVIEW

File: browser/views/index.pug [hunk 1]
```diff
@@ -31,14 +31,22 @@ block body
...
+            button.send-attachment(no-js='', on-click-toggle-class='.upload hide')
+              img(src='img/clip.png', width='25px')
+            input.file-input.hide(type='file')
             button.open-emoji(no-js='', on-click-toggle-class='.emojis hide')
               img(src='img/smiley.png', width='20px')
             .emojis.hide
               .emojis-header
               .emojis-body
+            .upload.hide 
+              button.attachment-items(name='image' title=".jpeg/.jpg")
```
Comment (ifedapoolarewaju) [inline browser/views/index.pug:43]: for semantics, I think we should rename this class to `attachment-item`

File: browser/js/funcs.js [hunk 2]
```diff
@@ -180,24 +180,43 @@ function removeSubmitHandler () {
...
 
 function addAttachmentSender (chat_) {
-  document.querySelector('.send-attachment').onclick = () => {
-    const fileInput = document.querySelector('.file-input');
-    fileInput.click();
-    fileInput.onchange = () => {
-      sendAttachment(fileInput.files[0].path, chat_);
-      fileInput.value = '';
+  const elements = document.querySelectorAll('.attachment-items');
+  const upload = document.querySelectorAll('.upload')[0];
```
Comment (ifedapoolarewaju) [inline browser/js/funcs.js:206]:
```suggestion
  const upload = document.querySelector('.upload');
```

File: main/instagram.js [hunk 3]
```diff
@@ -177,6 +177,41 @@ exports.uploadFile = function (filePath, recipients) {
       }).then(resolve).catch(reject);
     }).catch(reject);
   });
+}
+
+
```
Comment (ifedapoolarewaju) [inline main/instagram.js:182]: remove extra space?

File: main/instagram.js [hunk 4]
```diff
@@ -177,6 +177,41 @@ exports.uploadFile = function (filePath, recipients) {
...
+    const directThread = igClient.entity.directThread(recipients);
+    readFileAsync(filePath).then((buffer) => {
+      directThread.broadcastVoice({
+        file: buffer,
+      }).then(resolve).catch(reject);
+    }).catch(reject);
+  });
+}
+
+exports.uploadFile = function (filePath, _fileType, recipients) {
```
Comment (ifedapoolarewaju) [inline main/instagram.js:205]: is there a reason the variable `_fileType` isn't named `fileType` instead? 🤔

File: main/instagram.js (hunk 3 above)
Comment (vineetvk01) [reply main/instagram.js]: Yup :clapper:

**Comment by ifedapoolarewaju** [comment]
thank you!

**Pull request merged**